_data/simple-icons.json

```diff
@@ -14835,6 +14835,17 @@
             "hex": "000000",
             "source": "https://society6.com"
         },
+        {
+            "title": "Socket.dev",
+            "hex": "C93CD7",
+            "source": "https://socket.dev",
+            "aliases": {
+                "aka": [
+                    "Socket",
+                    "Socket Security"
+                ]
+            }
```

Comment:
These aliases are not well-known aliases. Please consider removing them.

Reply:
I removed them.